hack/create-metering-manifests.sh

```diff
@@ -39,43 +40,43 @@ trap 'rm -rf $TMPDIR' EXIT SIGINT
 CHART="$ROOT_DIR/charts/metering-ansible-operator"
 
 helm template "$CHART" \
-    "${VALUES_ARGS[@]}" \
+    ${VALUES_ARGS[@]+"${VALUES_ARGS[@]}"} \
```

Comment:
I'm not sure I understand this, I took a look at [indexing array by the @ character](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion), so wouldn't `VALUES_ARGS` always be empty?

Reply:
I should've viewed the entire file, I didn't see the while loop on line 28 - nvm

Reply:
Also read the commit message.